### ISSUE TITLE
Send logs to separate container

### ISSUE THREAD

**Issue by skamoen** [AreaFiftyLAN/lancie-api#494]
Current infrastructure doesn't persist logs over deployments to allow for smooth upgrades. However, we do want to keep logs, so let's see how we can send logs to a separate container with a persistent volume to collect our logs.

**Comment by dsluijk** [comment]
They are being written to a log file, so this is not needed.

**Comment by Fastjur** [comment]
No, that is not how it works. The logs are saved to a file inside the container, which will disappear if the container is rebooted, so they still need to be sent to a container with a persistent volume to ensure logs are not deleted whenever this container is rebooted.
